backend/private-graph/graph/resolver.go

```diff
@@ -2597,6 +2652,49 @@ func (r *Resolver) CreateHeightTaskAndAttachment(
 	return nil
 }
 
+func (r *Resolver) CreateJiraTaskAndAttachment(
+	ctx context.Context,
+	workspace *model.Workspace,
+	attachment *model.ExternalAttachment,
+	issueTitle string,
+	issueDescription string,
+	projectId string,
+) error {
+	accessToken, err := r.IntegrationsClient.GetWorkspaceAccessToken(ctx, workspace, modelInputs.IntegrationTypeJira)
+
+	if err != nil {
+		return err
+	}
+
+	if accessToken == nil {
+		return errors.New("No Jira integration access token found.")
+	}
+
+	jiraIssuePayload := jira.JiraCreateIssueFields{
+		Description: issueDescription,
+		Summary:     issueTitle,
+		Project:     jira.JiraIssueProjectData{Id: projectId},
+		IssueType:   jira.JiraIssueTypeData{Id: "10001"},
```

Comment:
I am not comfortable with this but have good reason to believe this default issue type comes with every jira project
and I haven't seen a way to delete it.

Reply:
I think this sounds good to me for a first version - longer term it might make sense to have a "task type" dropdown as part of the integration settings page or something like that